web_frontend/osmaxx/countries/_settings.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from django.conf import settings
+
+POLYFILE_LOCATION = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data', 'polyfiles')
+
+# only for testing, not documented
```

Comment:
That this is left out of the documentation (be it in the Readme or elsewhere) on purpose. But I agree, this can just be left out, it's nothing that helps to understand the code.